README.md

```diff
@@ -57,3 +57,64 @@ import Auth from '@heycharles/browser-sdk/dist/lib/Auth'
 ## LICENSE
 
 Apache-2.0
+
+
+### What to do with the Browser SDK
+
+The SDK is an API Abstraction layer between the backend and the UI. We must avoid putting business logic inside the SDK. We should use it to ensure parity between entities on the backend and the SDK to ensure a good developer experience.
+
+Example of abstraction:
+
+```ts
+export default { 
+   fetchCount: async (options?: UniverseFetchOptions): Promise<{ count: number }> => {
+        try {
+          const opts = {
+            method: 'HEAD',
+            url: `${this.universeBase}/${Feeds.endpoint}`,
+            params: {
+              ...(options?.query ?? {})
+            }
+          }
+
+          const res = await this.http.getClient()(opts)
+
+          return {
+            count: Number(res.headers['X-Resource-Count'] || res.headers['x-resource-count'])
+          }
+        } catch (err) {
+          throw new FeedFetchCountRemoteError(undefined, { error: err })
+        }
+      },
+}
+
+```
+
+### What not to do with the browser
```

Comment:
Yeah I think we can leave this one out completely. Someone could miss the "not" in the title and just scan the page for examples and see this.

If you really want some bad examples, I'd suggest to list them as bullet points. Liek

- Avoid adding business logic that is beyond making requests to the API and transforming the data we get as response
- Avoid adding wrapper methods that call existing methods with some specific parameters (e.g. you have `fetchCount` and you add `fetchSpecificCount` that calls `fetchCount({ specific: true })` inside)

Additionally, this is a public repo. Readme is for the consumer of the sdk, not for the developer. For contributors, you can use CONTRIBUTING.md